checkQC/app.py

```diff
@@ -1,22 +1,22 @@
 
-import yaml
 import sys
 
 from checkQC.qc_engine import QCEngine
+from checkQC.config import get_config
+from checkQC.run_type_recognizer import RunTypeRecognizer
 
 
 def start():
 
-    with open("config/config.yaml") as stream:
-        config = yaml.load(stream)
+    config = get_config("config/config.yaml")
 
-    machine_type = "hiseq2500_rapid"
-    read_length = '100-120'
-    run_mode = "paired_end"
+    runfolder = "./tests/resources/MiSeqDemo"
 
-    handler_config = config[machine_type][read_length][run_mode]["handlers"]
+    run_type_recognizer = RunTypeRecognizer(config=config, runfolder=runfolder)
+    instrument_type = run_type_recognizer.instrument_type()
+    read_length = run_type_recognizer.read_length()
```

Comment:
Yes. You are right, but I think it is something that can be left for later when we integrate the whole thing with the "real" config. There are some things that we need to figure out here, such as how to handle when we've got differing read lengths on read 1 and read 2 etc. If you're ok with it perhaps we can merge now, and then we can fix it later?